packages/dds/matrix/src/matrix.ts

```diff
@@ -429,7 +488,14 @@ export class SharedMatrix<T = any>
 		);
 		builder.addBlob(
 			SnapshotPath.cells,
-			serializer.stringify([this.cells.snapshot(), this.pending.snapshot()], this.handle),
+			serializer.stringify(
+				[
+					this.cells.snapshot(),
+					this.pending.snapshot(),
+					this.cellLastWriteTracker.snapshot(),
```

Comment:
That's a lot of data to serialize extra!
I'd expect that somewhere we will do GC of that data. I.e. at any moment in time (here, or on any other path) we should be able to clear info if it's outside of collab window.
I'd create some function to be able to do such GC, and call it from here, and maybe call periodically on other call paths.
Even better would be to have capability do it incrementally on other code paths, as scanning whole table is expensive (unless it's really sparse and proportional to number of user edits within collab window - in such case we can scan it on every change).

Reply:
I will make this improvement later.

Reply:
Please open a task for it. We should address it soonish...

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/6434